docs/reference/modules/cluster/remote-clusters-connect.asciidoc

```diff
@@ -0,0 +1,219 @@
+      "remote" : {
+        "cluster_one" : {    <1>
+          "seeds" : [
+            "127.0.0.1:9300" <2>
+          ]
+        }
+      }
+    }
+  }
+}
+----
+// TEST[setup:host]
+// TEST[s/127.0.0.1:9300/\${transport_host}/]
+<1> The cluster alias of this remote cluster is `cluster_one`.
+<2> Specifies the hostname and transport port of a seed node in the remote
+    cluster.
+
+You can use the <<cluster-remote-info,remote cluster info API>> to verify that
+the local cluster is successfully connected to the remote cluster:
+
+[source,console]
+----
+GET /_remote/info
+----
+// TEST[continued]
+
+The API response indicates that the local cluster is connected to the remote
+cluster with the cluster alias `cluster_one`:
+
+[source,console-result]
+----
+{
+  "cluster_one" : {
+    "seeds" : [
+      "127.0.0.1:9300"
+    ],
+    "connected" : true,
+    "num_nodes_connected" : 1,  <1>
+    "max_connections_per_cluster" : 3,
+    "initial_connect_timeout" : "30s",
+    "skip_unavailable" : false, <2>
+    "mode" : "sniff"
+  }
+}
+----
+// TESTRESPONSE[s/127.0.0.1:9300/$body.cluster_one.seeds.0/]
+// TEST[s/"connected" : true/"connected" : $body.cluster_one.connected/]
+// TEST[s/"num_nodes_connected" : 1/"num_nodes_connected" : $body.cluster_one.num_nodes_connected/]
+<1> The number of nodes in the remote cluster the local cluster is
+connected to.
+<2> Indicates whether to skip the remote cluster if searched through {ccs} but
+no nodes are available.
+
+[[configure-remote-clusters-dynamic]]
+==== Dynamically configure remote clusters
+Use the <<cluster-update-settings,cluster update settings API>> to dynamically
+configure remote settings on every node in the cluster. The following request
+adds three remote clusters: `cluster_one`, `cluster_two`, and `cluster_three`.
+
+The `seeds` parameter specifies the hostname and
+<<transport-settings,transport port>> (default `9300`) of a seed node in the 
+remote cluster.
+
+The `mode` parameter determines the configured connection mode, which defaults
+to <<sniff-mode,`sniff`>>. Because `cluster_one` doesn't specify a `mode`, it
+uses the default. Both `cluster_two` and `cluster_three` explicitly use
+different modes.
+
+[source,console]
+----
+PUT _cluster/settings
+{
+  "persistent": {
+    "cluster": {
+      "remote": {
+        "cluster_one": {
+          "seeds": [
+            "127.0.0.1:9300"
+          ]
+        },
+        "cluster_two": {
+          "mode": "sniff",
+          "seeds": [
+            "127.0.0.1:9301"
+          ],
+          "transport.compress": true,
+          "skip_unavailable": true
+        },
+        "cluster_three": {
+          "mode": "proxy",
+          "proxy_address": "127.0.0.1:9302"
+        }
+      }
+    }
+  }
+}
+----
+// TEST[setup:host]
+// TEST[s/127.0.0.1:9300/\${transport_host}/]
+
+You can dynamically update other cluster settings for a cluster, such as
+<<transport-settings,transport settings>>. The following request updates the
```

Comment:
```suggestion
You can dynamically update settings for a remote cluster after the initial configuration. The following request updates the
```

Not all transport settings in the linked doc page can be set for a remote cluster. For example, I don't think you can set `transport.host` or `transport.publish_port` for a remote cluster. So it feels wrong to just link the whole doc page. At the same time, most connection related transport settings should be applicable. If we do want the link, we should also add a brief statement about the limitation.